give.spec

```diff
@@ -1,63 +1,40 @@
-# The default behavior of rpmbuild is to strip binaries after installation
-# in order to reduce file size. The two macros defined below override that
-# behavior and compresses the man and info pages for debugging purposes.
+# add --with checks option (default is enable non strict checks)
+%bcond_with checks
+
+# The default behavior of rpmbuild, depending on your OS, often is to strip binaries
+# after installation in order to reduce file size. The two macros defined below
+# override that behavior and compresses the man and info pages for debugging purposes.
+# For more information: https://www.redhat.com/archives/rpm-list/2001-November/msg00257.html
 %define __os_install_post /usr/lib/rpm/brp-compress
 %define debug_package %{nil}
 
+%define default_give_dir /usr/give
+
```

Comment:
I recommend changing this to `give_dir` and wrapping the definition with a check. This way if someone wants to define give_dir to something else they can do so with `rpmbuild --define give_dir /foo/bar`.

If would look something like:

```
[...]
if %{!?give_dir:define give_dir /usr/give}
[...]
```
(note: you will want to test both `rpmbuild -ba give.spec` and `rpmbuild -ba --define give_dir /var/tmp/give` to confirm)